### PR TITLE
zh: update docs about data mount path for DM-worker

### PR DESCRIPTION
**Pull request by csuzhangxc**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

update docs about data mount path for DM-worker

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@DanielZhangQD @qiancai PTAL

File: zh/enable-tls-for-dm.md [hunk 1]
```diff
@@ -589,6 +590,15 @@ spec:
         ssl-ca: /var/lib/source-tls/${tidb_secret_name}/ca.crt
         ssl-cert: /var/lib/source-tls/${tidb_secret_name}/tls.crt
         ssl-key: /var/lib/source-tls/${tidb_secret_name}/tls.key
+
+    mysql-instances:
+      -
```
Comment (DanielZhangQD) [inline zh/enable-tls-for-dm.md:595]: Should combine this with the next line?

Comment (csuzhangxc) [reply zh/enable-tls-for-dm.md]: https://github.com/pingcap/docs-tidb-operator/pull/1218/commits/d8810e9364e6c112c64c7ec60b3471bbdcf4e59e

File: zh/enable-tls-for-dm.md [hunk 2]
```diff
@@ -589,6 +590,14 @@ spec:
         ssl-ca: /var/lib/source-tls/${tidb_secret_name}/ca.crt
         ssl-cert: /var/lib/source-tls/${tidb_secret_name}/tls.crt
         ssl-key: /var/lib/source-tls/${tidb_secret_name}/tls.key
+
+    mysql-instances:
+      - source-id: "replica-01"
+        loader-config-name: "global"
```
Comment (DanielZhangQD) [inline zh/enable-tls-for-dm.md:596]:
```suggestion
    mysql-instances:
    - source-id: "replica-01"
       loader-config-name: "global"
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: zh/deploy-tidb-dm.md [hunk 3]
```diff
@@ -206,16 +206,20 @@ spec:
 
 2. 填写 `source1.yaml` 的 `from.host` 为 Kubernetes 集群内部可以访问的 MySQL host 地址。
 
-3. `source1.yaml` 文件准备好后，通过 `/dmctl --master-addr ${dm_cluster_name}-dm-master:8261 operate-source create source1.yaml` 将 MySQL-1 的数据源加载到 DM 集群中。
+3. 填写 `source1.yaml` 的 `relay-dir` 为持久卷在 Pod 内挂载目录 `/var/lib/dm-worker` 下的子目录，如 `/var/lib/dm-worker/relay`。
 
-4. 对 MySQL-2 及其他数据源，采取同样方式修改配置文件中的相关信息，并执行相同的 dmctl 命令。
+4. `source1.yaml` 文件准备好后，通过 `/dmctl --master-addr ${dm_cluster_name}-dm-master:8261 operate-source create source1.yaml` 将 MySQL-1 的数据源加载到 DM 集群中。
+
+5. 对 MySQL-2 及其他数据源，采取同样方式修改配置文件中的相关信息，并执行相同的 dmctl 命令。
```
Comment (qiancai) [inline zh/deploy-tidb-dm.md:213]:
```suggestion
5. 对 MySQL-2 及其他数据源，采取同样方式填写数据源 `yaml` 文件中的相关信息，并执行 dmctl 命令将对应的数据源加载到 DM 集群中。
```

**Review by qiancai — COMMENTED** [review]
@csuzhangxc PTAL

**Pull request merged**